### PR TITLE
Check for wasm-pack

### LINKED ISSUE #22: Check if `wasm-pack` is available in $PATH
We should check it at the beginning and print a user-friendly error in case it doesn't exists.

### PR DESCRIPTION
**Pull request by sendilkumarn**
fix #22 

r? @xtuc

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,5 +29,8 @@
     "chalk": "^2.4.1",
     "glob": "^7.1.2",
     "watchpack": "^1.6.0"
+  },
+  "devDependencies": {
```
[the remark this answers is not in the document]
Comment (sendilkumarn) [reply package.json]: You are correct. 👍

**Comment by sendilkumarn** [comment]
Yes, we can remove that.

**Comment by xtuc** [comment]
Feel free to merge this when you are ready. I can release it.

**Comment by sendilkumarn** [comment]
@xtuc slightly refactored things. It is good to merge now

**Pull request merged**